packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -578,10 +578,10 @@ To use it, simply create one:
 
 ```ts
 // This will provide Docker container images for the latest Deadline release
-const images = new ThinkboxDockerImages(scope, 'Images', {
-  // Change this to AwsThinkboxEulaAcceptance.USER_ACCEPTS_AWS_THINKBOX_EULA to accept the terms
-  // of the AWS Thinkbox End User License Agreement
-  userAwsThinkboxEulaAcceptance: AwsThinkboxEulaAcceptance.USER_REJECTS_AWS_THINKBOX_EULA,
+const images = new ThinkboxDockerImages(stack, 'Image', {
+  // Change this to AwsCustomerAgreementAndIpLicenseAcceptance.USER_ACCEPTS_AWS_CUSTOMER_AGREEMENT_AND_IP_LICENSE to accept the terms
+  // of the AWS Customer Agreement and AWS Intellectual Property License.
+  userAwsCustomerAgreementAndIpLicenseAcceptance: AwsCustomerAgreementAndIpLicenseAcceptance.USER_ACCEPTS_AWS_CUSTOMER_AGREEMENT_AND_IP_LICENSE,
```

Comment:
Default here should be rejects as well (applies to rest of this file)

Reply:
Oh no! Really good eye! How did I make that mistake?!